tests/files/drupal9-mariadb-single/assets/lagoon.site.yml

```diff
@@ -1,8 +1,8 @@
 '*':
-  host: lagoon-core-ssh.lagoon.svc
+  host: lagoon-core-ssh-token.lagoon.svc
```

Comment:
this needs to be the `lagoon-remote-ssh-portal`

Reply:
ok trying again